src/js/constants.js

```diff
@@ -57,6 +58,9 @@ export const MediaUIProps = {
   MEDIA_STREAM_TYPE: 'mediaStreamType',
   MEDIA_TARGET_LIVE_WINDOW: 'mediaTargetLiveWindow',
   MEDIA_TIME_IS_LIVE: 'mediaTimeIsLive',
+  MEDIA_RENDITION_LIST: 'mediaRenditionList',
+  MEDIA_RENDITION_ENABLED: 'mediaRenditionEnabled',
+  MEDIA_RENDITION_ACTIVE: 'mediaRenditionActive',
```

Comment:
yes, when the rendition is enabled / disabled it's not immediately playing because a stream switch can take a bit.
when the rendition starts playing then it becomes active...